templates/service.j2

```diff
@@ -45,7 +45,13 @@ After={{ systemd_service_Unit_After | join(",") }}
 
 [Service]
 ExecStart={{ systemd_service_Service_ExecStart }}
+{% if systemd_service_Environment_File is defined %}
+{% if systemd_service_Environment_File is string %}
+EnvironmentFile={{ systemd_service_Environment_File }}
+{% else %}
 EnvironmentFile=-{{ systemd_service_default_dir }}/{{ systemd_service_name }}
```

Comment:
My question is regarding "**-**" : _The argument passed should be an absolute filename or wildcard expression, optionally prefixed with "-", which indicates that if the file does not exist, it will not be read and no error or warning message is logged._

I would remove from both so if we forget to create env_vars this will fail, will be logged and we can debug.

Reply:
mmh good point although it is tangential to this PR, but I could push it in another commit

Reply:
I mean, in another PR with a different version tag because this could be a breaking change